apps/src/templates/watchers/Watchers.jsx

```diff
@@ -462,3 +475,26 @@ function nonValueDescriptor(obj) {
 function wrapValue(index, length) {
   return (index + length) % length;
 }
+
+function parseArray(array) {
+  let parsedArray = '';
+  array.forEach((element, index, array) => {
+    if (element === null) {
+      parsedArray = parsedArray + 'null';
+    } else if (Array.isArray(element)) {
+      parsedArray = parsedArray + 'list (' + element.length + ')';
```

Comment:
Should/could this recursively parse so that watching a list of lists would show you the contents of all the lists?

Reply:
Also interested in this^?

Reply:
I decided not to for a couple reasons:
1. Students could potentially break applab that way
```
myArray = [1,2]
firstElement = [1]
while(1>0) {
  firstElement[0] = myArray
  firstElement = firstElement[0]
}
```
^ Although in retrospect, this would probably break applab anyway.

2. It starts to look pretty confusing when you have multiple arrays or multiple levels of arrays

3. I took a look at chrome to see what they do, and while Chrome _does_ have a way to access elements in an array in its debugger, it doesn't display all the elements by default (the exception to this is if you have only one level of nested arrays i.e. myArray = [1, [2, 3], 4])
Chrome example:
![image](https://user-images.githubusercontent.com/8324574/54848153-c0384380-4c9d-11e9-93c8-89de9b08570c.png)

Reply:
I decided not to for a couple reasons:
1. It starts to look confusing when you have multiple lists or multiple levels of nesting in a list. The way our UI currently is set up, you end up with a bunch of commas, and it's difficult to figure out which array each element belongs to.

2. I checked how the chrome debugger handles this case, and it does something similar. It doesn't expand nested arrays by default. (the exception to this is it will if there is only a single level of nesting)
Chrome example:
![image](https://user-images.githubusercontent.com/8324574/54849046-4ce40100-4ca0-11e9-81ce-b7b2ebcdb366.png)